go.mod

```diff
@@ -4,7 +4,7 @@ module storj.io/storj
 require (
 	github.com/btcsuite/btcutil v0.0.0-20180706230648-ab6388e0c60a
 	github.com/garyburd/redigo v1.0.1-0.20170216214944-0d253a66e6e1 // indirect
-	github.com/graphql-go/graphql v0.7.6
+	github.com/graphql-go/graphql v0.7.8
```

Comment:
Use `go get -u github.com/graphql-go/graphql@master` to update it and then you can remove `storjql` package.